jenkins/dev_env/build_prerequisite.sh

```diff
@@ -42,6 +42,7 @@ ls /var/lib/ha_env/ | grep yum_init || {
 yum install -y gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel python36-dbus  --nogpgcheck
 yum group -y install "Development Tools" --nogpgcheck
 
+# TODO below requiremnt installation (python_requirements.txt, python_requirements.ext.txt) can be removed.
```

Comment:
Why we need to remove it?

Reply:
@Madhura-08 We are not using any third party but utils.
and cortx utils will install its all prerequisites.
and why keep a dependency on versions as every time we need to take the case or this when utils update it.
utils itself needs to keep track of all dependency.
so needs to remove the below requirements.
So please let me know if we need it.
As discussed @ArchanaLimaye that if this can be removed later the only thing, I can comment on this as TODO so I have added.

Reply:
Ok. Sure. Got it.